distro/run/pom.xml

```diff
@@ -41,6 +41,12 @@
         <version>${version.graal.js}</version>
       </dependency>
 
+      <dependency>
```

Comment:
📓 No action, just note. In the past we always put the overrides before the spring dependencies but this doesn't seem to be needed anymore `9.37.4` is picked up correctly.